.bazelci/presubmit.yml

```diff
@@ -4,17 +4,6 @@ x_defaults:
   # it doesn't know about; so that is used to avoid repeating common subparts.
   common: &common
     platform: macos_arm64
-    shell_commands:
-    - xcrun simctl runtime list
-    - xcrun simctl runtime delete 22D8075 || true
-    - xcrun simctl runtime delete 21F79 || true
-    - xcrun simctl runtime list
-    - xcrun simctl runtime match list
-    - xcrun simctl runtime match set xros2.2 22N895
```

Comment:
https://buildkite.com/bazel/rules-apple-darwin/builds/10357#01967ee4-5ba5-4352-a6d4-b63973b2ff6f

@keith Looks like xros2.2 has to be updated to some new value?